packages/flame/lib/src/events/messages/tap_down_event.dart

```diff
@@ -44,4 +44,9 @@ class TapDownEvent extends PositionEvent {
       ),
     );
   }
+
+  @override
+  String toString() =>
+      'TapDownEvent(pointerId;$pointerId, deviceKind:$deviceKind, '
```

Comment:
```suggestion
      '$TapDownEvent(pointerId;$pointerId, deviceKind:$deviceKind, '
```
Will help in keeping class name in this string in-sync with the actual class name in future as well.